src/repositories/connectToDB.ts

```diff
@@ -25,6 +25,7 @@ async function prepareDatabase() {
     { token: 1, ballot: 1 },
     { unique: true }
   );
+  await getCollection("api-key").createIndex({ name: 1 }, { unique: true });
```

Comment:
Please also add the `key` field as part of the collection-index as it's being used to check the existence of incoming requests with api-keys:
https://github.com/LeoTuet/backend-we-want-to-sleep/blob/15d45eb20491be6fc2ddf5ab5c04687d4644f497/src/handler/AuthHandler.ts#L30-L33